detection_rules/devtools.py

```diff
@@ -435,6 +437,18 @@ def integrations_pr(ctx: click.Context, local_repo: str, token: str, draft: bool
                     pkg_directory: str, base_branch: str, remote: str,
                     branch_name: Optional[str], github_repo: str, assign: Tuple[str, ...], label: Tuple[str, ...]):
     """Create a pull request to publish the Fleet package to elastic/integrations."""
+    github = GithubClient(token)
+    github.assert_github()
+    client = github.authenticated_client
+    repo = client.get_repo(github_repo)
+
+    # Use elastic-package to format and lint
+    gopath = utils.gopath()
+    assert gopath is not None, "$GOPATH isn't set"
+
+    err = 'elastic-package missing, run: go install github.com/elastic/elastic-package@latest'
+    assert subprocess.check_output(['elastic-package'], stderr=subprocess.DEVNULL), err
```

Comment:
This may throw a `FileNotFoundError` if the `elastic-package` is not installed, before the assert is evaluated. 
```suggestion
        try:
            subprocess.check_output(['elastic-package'], stderr=subprocess.DEVNULL)
        except FileNotFoundError:
            raise Exception(err)
```

Reply:
I know but in this case I was content with letting that pass through since it is self explanatory. This was meant to also detect that it is in the user path. I will update the error to reflect.